### PR TITLE
Add vic-machine create integration tests

### PR DESCRIPTION
**Pull request by andrewtchin**

### CODE REVIEW

**Comment by emlin** [comment]
lgtm

File: tests/resources/Util.robot [hunk 1]
```diff
@@ -11,30 +11,30 @@ Library  DateTime
 ${bin-dir}  /drone/src/github.com/vmware/vic/bin
 
 *** Keywords ***
-Install VIC Appliance To Test Server
+Set Test Environment Variables
     [Arguments]  ${certs}=${false}  ${vol}=default
```
Comment (mhagen-vmware) [inline tests/resources/Util.robot]: I am not a big fan of double layering the default options here... under the typical scenario it gets set correctly by the defaults from the intall vic appliance keyword, but doubling them up here, causes duplication of code maintenance. Can we remove this and just put them in your test cases?

File: tests/test-cases/Group6-VIC-Machine/6-4-Create-Basic.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+```
+5. Run regression tests
+6. Remove the VCH
+7. Issue the following command:
+```
+vic-machine-linux create --name=<VCH_NAME> --target=<TEST_URL> --user=<TEST_USERNAME> --image-datastore=<TEST_DATASTORE> --appliance-iso=bin/appliance.iso --bootstrap-iso=bin/bootstrap.iso --password=<TEST_PASSWORD> --force=true --bridge-network=network --compute-resource=<TEST_RESOURCE> --timeout <TEST_TIMEOUT> --volume-store=<TEST_DATASTORE>/test:default
+```
+8. Run regression tests
+9. Remove the VCH
+
```
Comment (mhagen-vmware) [inline tests/test-cases/Group6-VIC-Machine/6-4-Create-Basic.md]: Can we get a couple of negative test cases here? At least a few that just attempt to supply and invalid resource? Like a bad compute-resource, or bad bridge-network and if there is any required param try not supplying it, and supplying a parameter that doesn't exist.

File: tests/test-cases/Group6-VIC-Machine/6-4-Create-Basic.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+#Expected Outcome:
+* Step 1 should install the VCH
+* Step 2 should successfully run the regression tests
```
Comment (mhagen-vmware) [inline tests/test-cases/Group6-VIC-Machine/6-4-Create-Basic.md]: Steps 2, 5, 8

File: tests/test-cases/Group6-VIC-Machine/6-4-Create-Basic.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+* Step 3 should remove the VCH
```
Comment (mhagen-vmware) [inline tests/test-cases/Group6-VIC-Machine/6-4-Create-Basic.md]: Step 3,6,9

File: tests/test-cases/Group6-VIC-Machine/6-4-Create-Basic.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+6. Remove the VCH
+7. Issue the following command:
+```
+vic-machine-linux create --name=<VCH_NAME> --target=<TEST_URL> --user=<TEST_USERNAME> --image-datastore=<TEST_DATASTORE> --appliance-iso=bin/appliance.iso --bootstrap-iso=bin/bootstrap.iso --password=<TEST_PASSWORD> --force=true --bridge-network=network --compute-resource=<TEST_RESOURCE> --timeout <TEST_TIMEOUT> --volume-store=<TEST_DATASTORE>/test:default
+```
+8. Run regression tests
+9. Remove the VCH
+
+#Expected Outcome:
+* Step 1 should install the VCH
```
Comment (mhagen-vmware) [inline tests/test-cases/Group6-VIC-Machine/6-4-Create-Basic.md]: Step 1,4,7

File: tests/test-cases/Group6-VIC-Machine/6-4-Create-Basic.robot [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+*** Test Cases ***
+Create VCH - defaults
+    Log To Console  \nRunning vic-machine create - defaults
+    Set Test Environment Variables
+    Set Test VCH Name
+    ${output}=  Run  bin/vic-machine-linux create --name=${vch-name} --target=%{TEST_URL} --user=%{TEST_USERNAME} --image-datastore=%{TEST_DATASTORE} --password=%{TEST_PASSWORD}
+    Run Keyword And Ignore Error  Should Contain  ${output}  Installer completed successfully
+    Get Docker Params  ${output}
+    Sleep  10 seconds
```
Comment (mhagen-vmware) [inline tests/test-cases/Group6-VIC-Machine/6-4-Create-Basic.robot]: Can you put the github tracker in here, so when/if that issue ever gets fixed we don't have to sleep for 30+ seconds for this test anymore?

File: tests/test-cases/Group6-VIC-Machine/6-4-Create-Basic.robot [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+    Set Test Environment Variables
+    Set Test VCH Name
+    ${output}=  Run  bin/vic-machine-linux create --name=${vch-name} --target="%{TEST_USERNAME}:%{TEST_PASSWORD}@%{TEST_URL}" --image-datastore=%{TEST_DATASTORE}
+    Run Keyword And Ignore Error  Should Contain  ${output}  Installer completed successfully
+    Get Docker Params  ${output}
+    Sleep  10 seconds
+    Run Regression Tests
+    Cleanup VIC Appliance On Test Server
+
+Create VCH
```
Comment (mhagen-vmware) [inline tests/test-cases/Group6-VIC-Machine/6-4-Create-Basic.robot]: Create VCH - full params?

File: tests/test-cases/Group6-VIC-Machine/6-4-Create-Basic.robot (hunk 6 above)
Comment (andrewtchin) [inline tests/test-cases/Group6-VIC-Machine/6-4-Create-Basic.robot]: fixed

**Comment by jzt** [comment]
lgtm 🍧 🉐

**Comment by andrewtchin** [comment]
♻️

**Comment by andrewtchin** [comment]
♻️

**Pull request merged**